src/components/Map/Map.js

```diff
@@ -1,24 +1,46 @@
-import { useState, useEffect } from "react";
-import { createMap, setCurrentLocationAsCenter } from "services/MapService";
+import { useState, useEffect } from 'react';
+import { GoogleMap, useLoadScript } from '@react-google-maps/api';
 import "components/Map/Map.scss";
 
-const MAP_ID = "map";
+const defaultLocation = { lat: 19.432, lng: -99.133 };
+const defaultZoom = 12;
 
 const Map = () => {
-  const [map, setMap] = useState(null);
+  const [currentLocation, setCurrentLocation] = useState(defaultLocation);
+  
+  const { isLoaded, loadError } = useLoadScript({
+    googleMapsApiKey: process.env.REACT_APP_GOOGLE_MAPS_API_KEY
+  });
 
   useEffect(() => {
-    const initializeMap = async () => {
-      setMap(await createMap("map"));
-      setCurrentLocationAsCenter(map);
-    };
-
-    initializeMap();
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        setCurrentLocation({
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        });
+      });
+    }
```

Comment:
Looks a bit cleaner IMO
```suggestion
    navigator.geolocation?.getCurrentPosition(({ coords }) => {
      setCurrentLocation({
        lat: coords.latitude,
        lng: coords.longitude
      });
    });
```